ts/services/ElementService.ts

```diff
@@ -299,6 +299,87 @@ class ElementService {
     };
   }
 
+  public async customtypeImageRm(imageName, config, logger): Promise<void> {
```

Comment:
Now that these are not prefixed by element can you move them to a new file `ImageService.ts`.